src/signers/Web3Signer.ts

```diff
@@ -98,4 +100,25 @@ export class Web3Signer implements TLSigner {
     const msgHashBytes = ethers.utils.arrayify(msgHash)
     return this.signMessage(msgHashBytes)
   }
+
+  public async getTxInfos(userAddress: string): Promise<TxInfos> {
```

Comment:
How did you test this part of the code?

Reply:
Not tested, I thought we'd have tests with the making a transaction with the Web3Signer already, and this would have been used by it.
I will try to add tests but am not sure how to create a web3signer object yet.

Reply:
I think at the moment we can only manually test that. Please ask Dongha

Reply:
I added a unit test that made explicit missing parenthesis, in this method. I'll ask Dong-Ha for manual testing.